src/odemis/gui/model/__init__.py

```diff
@@ -194,6 +195,8 @@ class MainGUIData(object):
         "ebeam-shift": "beamshift",
         "diagnostic-ccd": "ccd",
         "det-rotator": "det_rotator",
+        "se-detector-ion": "ion_sed",
+        "stage-global": "stage_global",
```

Comment:
You need to add these 2 variables to the __init__ too. (eg, line 273). This way, if the components are not present, they will be set to `None` (otherwise, the attributes are not present).